test/e2e/apimachinery/aggregator.go

```diff
@@ -382,14 +382,13 @@ func TestSampleAPIServer(f *framework.Framework, image string) {
 	flunderName = generateFlunderName("dynamic-flunder")
 
 	// Rerun the Create/List/Delete tests using the Dynamic client.
-	resources, err := client.Discovery().ServerPreferredNamespacedResources()
-	framework.ExpectNoError(err, "getting server preferred namespaces resources for dynamic client")
+	resources, discoveryErr := client.Discovery().ServerPreferredNamespacedResources()
```

Comment:
partial discovery is always a possibility, especially in the presence of rapidly changing API group registration (like in extension API / CRD tests). the thing we are about here is that our extension API group showed up